docs/tempo/website/api_docs/_index.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: API documentation
+title: Tempo API
 weight: 500
 ---
```

Comment:
Would you be able to add a richer set of Hugo metadata here?  A description and a menuTitle would be great.